### ISSUE TITLE
String scope isn’t starting properly in multiline array definition

### ISSUE THREAD

**Issue by ghost** [jfcherng/Sublime-ShellScriptImproved#23]
In a statement like this

``` bash
declare -A ERROR_MESSAGES=(
    [no msg]=''
    [unknown]='Unknown error happened.'
    [no util]="‘$util’ is required but wasn’t found on this system."
)
```

The value of the second array element isn’t matched properly, but it would if all the elements were passed in one row. 

![1470664324](https://cloud.githubusercontent.com/assets/2245254/17485392/a92266f8-5d9e-11e6-8f9f-fa3a79729fa9.png)

**Comment by jfcherng** [comment]
A reminder for me: http://www.artificialworlds.net/blog/2012/10/17/bash-associative-array-examples

**Comment by jfcherng** [comment]
This should be resolved in [1.2.10](https://github.com/jfcherng/Sublime-ShellScriptImproved/releases/tag/1.2.10).
